### PR TITLE
Feat/new jupyter doc and hierarcy

### PR DESCRIPTION
**Pull request by zwyll**
Updated the document for DesignSafe Jupyter HPC.

### CODE REVIEW

**Review by wesleyboar — COMMENTED** [review]
I like not using ` (backtick).

But, I suspect we should be using " (quotes) or nothing instead of ** (bold) and * (italic), but I defer to @susanunit's judgement.

File: user-guide/docs/tools/jupyterhub/jupyterlabhpc.md [hunk 1]
```diff
@@ -1,26 +1,35 @@
...
+1. Go to [DesignSafe website](https://www.designsafe-ci.org) and sign in to your DesignSage account. Then click *Use DesignSafe* > *Tools & Applications* > *Analysis* > *Jupyter* and select *Jupyter Lab HPC (GPU)*.
 
     ![DesignSafe Portal](imgs/DesignSafe_Portal.png)
 
-2. Once define your maximum Job runtime (no more than 48 hours) and Job name, click the `Run` buttton so the job will be submitted. Then you can monitor your job status on the right column.
+2. Once define your maximum Job runtime (no more than 48 hours) and Job name, click the *Run* buttton so the job will be submitted. Then you can monitor your job status on the right column.
 
-3. Once your job starts, the Job Status bar will show status as "RUNNING" and a small window will pop up. Then click the `CONNECT!` button to connect to your interactive session, i.e., JupyterLab.
+3. Once your job starts, the Job Status bar will show status as "RUNNING" and a small window will pop up. Then click the *CONNECT!* button to connect to your interactive session, i.e., JupyterLab.
```
Comment (wesleyboar) [inline user-guide/docs/tools/jupyterhub/jupyterlabhpc.md:25]: To match [private answer](https://tacc-team.slack.com/archives/C06S93KKA5R/p1713808703941429) from [private confirmation request](https://tacc-team.slack.com/archives/C06S93KKA5R/p1713196766860919) for desired punctuation:

```suggestion
3. Once your job starts, the Job Status bar will show status as "RUNNING" and a small window will pop up. Then click the "CONNECT!" button to connect to your interactive session, i.e., JupyterLab.
```

File: user-guide/docs/tools/jupyterhub/jupyterlabhpc.md [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
+| **Jupyter Lab HPC (GPU)** | Four GPUs in total, best performance for AI/ML jobs; <br> Many commonly used Python packages for AI/ML applications have been pre-installed, including TensorFlow and PyTorch | Queue exists, longer wait time may be needed; <br>Maximum job runtime is 48 hours | 
+
+**Note:** When using the **Jupyter Lab HPC (CPU)** and **Jupyter Lab HPC (GPU)**, you need to place all your files/data/codes/Jupyter Notebooks inside the *HPCWork* folder. This can be done on DesignSafe Data Depot page without submitting any jobs.
+
+   ![JupyterLab](imgs/HPCWork.png)
 
 ## Launch the Jupyter Lab HPC (GPU)
 
-1. Go to [DesignSafe website](https://www.designsafe-ci.org) and sign in to your DesignSage account. Then click `Use DesignSafe` --> `Tools & Applications` --> `Analysis` --> `Jupyter` and select `Jupyter Lab HPC (GPU)`.
+1. Go to [DesignSafe website](https://www.designsafe-ci.org) and sign in to your DesignSage account. Then click *Use DesignSafe* > *Tools & Applications* > *Analysis* > *Jupyter* and select *Jupyter Lab HPC (GPU)*.
```
Comment (wesleyboar) [inline user-guide/docs/tools/jupyterhub/jupyterlabhpc.md:19]: To match [private answer](https://tacc-team.slack.com/archives/C06S93KKA5R/p1713808703941429) from [private confirmation request](https://tacc-team.slack.com/archives/C06S93KKA5R/p1713196766860919) for desired punctuation:

```suggestion
1. Go to [DesignSafe website](https://www.designsafe-ci.org) and sign in to your DesignSage account. Then click "Use DesignSafe" > "Tools & Applications" > "Analysis" > "Jupyter" and select "Jupyter Lab HPC (GPU)".
```

File: user-guide/docs/tools/jupyterhub/jupyterlabhpc.md [hunk 3]
```diff
@@ -1,26 +1,35 @@
...
 
     ![DesignSafe Portal](imgs/DesignSafe_Portal.png)
 
-2. Once define your maximum Job runtime (no more than 48 hours) and Job name, click the `Run` buttton so the job will be submitted. Then you can monitor your job status on the right column.
+2. Once define your maximum Job runtime (no more than 48 hours) and Job name, click the *Run* buttton so the job will be submitted. Then you can monitor your job status on the right column.
```
Comment (wesleyboar) [inline user-guide/docs/tools/jupyterhub/jupyterlabhpc.md:23]: To match [private answer](https://tacc-team.slack.com/archives/C06S93KKA5R/p1713808703941429) from [private confirmation request](https://tacc-team.slack.com/archives/C06S93KKA5R/p1713196766860919) for desired punctuation:

```suggestion
2. Once define your maximum Job runtime (no more than 48 hours) and Job name, click the "Run" buttton so the job will be submitted. Then you can monitor your job status on the right column.
```

File: user-guide/docs/tools/jupyterhub/jupyterlabhpc.md [hunk 4]
```diff
@@ -1,26 +1,35 @@
...
-3. Once your job starts, the Job Status bar will show status as "RUNNING" and a small window will pop up. Then click the `CONNECT!` button to connect to your interactive session, i.e., JupyterLab.
+3. Once your job starts, the Job Status bar will show status as "RUNNING" and a small window will pop up. Then click the *CONNECT!* button to connect to your interactive session, i.e., JupyterLab.
 
     ![Job Status](imgs/Job_status.png)
 
 
 ## Launch the Jupyter Notebook
 
-1. Navigate to the `HPCWork` folder. You need to place all your data/codes/Jupyter Notebooks inside this folder. 
+1. Navigate to the *HPCWork* folder. You need to place all your data/codes/Jupyter Notebooks inside this folder.
```
Comment (wesleyboar) [inline user-guide/docs/tools/jupyterhub/jupyterlabhpc.md:32]: To match [private answer](https://tacc-team.slack.com/archives/C06S93KKA5R/p1713808703941429) from [private confirmation request](https://tacc-team.slack.com/archives/C06S93KKA5R/p1713196766860919) for desired punctuation:

```suggestion
1. Navigate to the "HPCWork" folder. You need to place all your `data/codes/Jupyter Notebooks` inside this folder. 
```

Also includes extra `bit of formatting`.

**Review by wesleyboar — CHANGES_REQUESTED** [review]
As discussed privately, let's use quotations for text form an interface.

**Review by wesleyboar — APPROVED** [review]

**Comment by cockerilltim** [comment]
Approved.

**Review by cockerilltim — APPROVED** [review]
approved

**Pull request merged**